docs/contributing.rst

```diff
@@ -94,6 +94,32 @@ further parameterize pytest for local testing.
 For all valid arguments, check `the pytest documentation
 <https://docs.pytest.org/en/stable/usage.html#stopping-after-the-first-or-n-failures>`_.
 
+Getting paid for your contributions
+-----------------------------------
+
+urllib3 has a `pool of money hosted on OpenCollective <https://opencollective.com/urllib3#category-BUDGET>`_
+which the team uses to pay contributors for their work. **That could be you, too!** If you close an issue
+that is marked with the `"Paid $$$" label <https://github.com/urllib3/urllib3/issues?q=is%3Aissue+is%3Aopen+label%3A%22Paid%20%24%24%24%22>`_ then you're eligible to be paid for your work.
+
+- Ensure that you're able to receive funds from OpenCollective.
```

Comment:
Does OpenCollective have any information about eligibility or setup we could link to?